boot/zephyr/prj.conf

```diff
@@ -33,3 +33,4 @@ CONFIG_FLASH=y
 CONFIG_LOG=y
 ### Ensure Zephyr logging changes don't use more resources
 CONFIG_LOG_DEFAULT_LEVEL=0
+CBPRINTF_NANO=y
```

Comment:
```suggestion
CONFIG_CBPRINTF_NANO=y
```

Reply:
@tejlmand Would it be too hard to enable the build system to prefix options with `CONFIG_` if it's not already there?

Reply:
fixed

I have been testing a few version of the modification before, so in finally i commit patch with obvious bug.
It is a pity that not possible `imply` Kconfig` bool property while it is a `choice` part.

Reply:
> @tejlmand Would it be too hard to enable the build system to prefix options with `CONFIG_` if it's not already there?

No, but I don't like to start anticipating what user intended if he forgets `CONFIG_`.

Could be a simple mistake like: `COFNIG_x` and then it would be wrong to prepend `CONFIG_` also.

After all, the following error is printed with invalid conf files:
```
<path>/prj.conf:2: warning: ignoring malformed line 'CBPRINTF_NANO=y'

error: Aborting due to Kconfig warnings
```